runtime/caches/redis.ts

```diff
@@ -47,13 +39,33 @@ export const caches: CacheStorage = {
   ): Promise<Response | undefined> => {
     throw new Error("Not Implemented");
   },
-  open: (cacheName: string): Promise<Cache> => {
-    if (!redis) {
-      throw new Error(
-        "Redis coult not be used due to the lack of credentials.",
-      );
-    }
-    const requestURLSHA1 = withCacheNamespace(cacheName);
+  open: async (_: string): Promise<Cache> => {
+    const redis = createClient({
+      url: REDIS_URL,
+    }).on("error", (err: Error) => {
+      console.error(err);
+
+      redis.disconnect();
+    }).on("connect", () => {
+      console.debug("[cache]", "connecting");
+    }).on("reconnecting", () => {
+      console.warn("[cache]", "reconnecting");
+    });
+
+    const ensureConnection = () => {
```

Comment:
ensure is returning boolean but I couldn't see any usages, can it just return void?